caliban/config/__init__.py

```diff
@@ -107,19 +107,21 @@ def expand_image(image: str) -> str:
 
 
 AptPackages = s.Or(
-    [str], {
-        s.Optional("gpu", default=list): [str],
-        s.Optional("cpu", default=list): [str]
-    },
+    s.Schema([str]),
```

Comment:
this is a problem. We definitely shouldn't have to wrap any compound data structure in Schema to get it to work with a formatting string. If we don't do this, we see:

https://github.com/keleshev/schema/issues/240